src/js/tabs/power.js

```diff
@@ -99,12 +99,12 @@ TABS.power.initialize = function (callback) {
             }
         }
 
-        var template = $('#tab-power-templates .voltage-configuration');
-        for (var index = 0; index < FC.VOLTAGE_METER_CONFIGS.length; index++) {
-            var destination = $('#voltage-meter-' + index + ' .configuration');
-            var element = template.clone();
+        template = $('#tab-power-templates .voltage-configuration');
```

Comment:
Reuse the template variable across the entire code maybe is not a good idea. I think is better and clearer to assign a different name for each of them, for example here `templateVoltageConfiguration`.
Caution with this change, not forget to rename where this template variable is used.
I think is only useful to reuse variable if the value is conditional